translations/messages.fr.xlf

```diff
@@ -2287,6 +2295,82 @@
+        <target>Vous ne pouvez pas sauter au dessus d'une même case plusieurs fois</target>
+      </trans-unit>
+      <trans-unit id="8833682869379041729" datatype="html">
+        <source>Cannot go backward with normal piece</source>
+        <target>Vous ne pouvez pas aller en arrière avec une pièce normale</target>
+      </trans-unit>
+      <trans-unit id="8649496248690252329" datatype="html">
+        <source>Cannot capture empty space (nor jump over it)</source>
+        <target>Vous ne pouvez pas capturer une case vide (ni sauter par dessus)</target>
+      </trans-unit>
+      <trans-unit id="8144038524408187730" datatype="html">
+        <source>Capture is mandatory</source>
+        <target>La capture est obligatoire</target>
+      </trans-unit>
+      <trans-unit id="4753584102674404468" datatype="html">
+        <source>You must finish this capture</source>
+        <target>Vous devez finir cette capture</target>
+      </trans-unit>
+      <trans-unit id="8274439231274366704" datatype="html">
+        <source>Lasca: origins</source>
+        <target>Lasca: les origines</target>
+      </trans-unit>
+      <trans-unit id="9130733749994426297" datatype="html">
+        <source>Lasca is a game based on draughts created in 1911 by Emanuel Lasker, chess world champion. It&apos;s played on a 7x7 board, each players has 11 pieces.</source>
+        <target>Lasca est un jeu basé sur le jeu de dames, crée en 1911 par Emanuel Lasker, champion du monde d'échecs. Lasca se joue sur un plateau de 7x7 cases, chaque joueur a 11 pièces.</target>
+      </trans-unit>
+      <trans-unit id="3198994967671406143" datatype="html">
+        <source>The goal of Lasca is, like for draughts, to put the opponent out of possible move, either by capturing all his pieces, either by blocking them.</source>
+        <target>Le but de Lasca est, comme aux dames, de rendre l'adversaire incapable de jouer, soit en capturant toutes ses pièces, soit en les bloquant</target>
+      </trans-unit>
+      <trans-unit id="1534367694207535197" datatype="html">
+        <source>Steps</source>
+        <target>Pas</target>
+      </trans-unit>
+      <trans-unit id="5328481068359252467" datatype="html">
+        <source>A simple step is made by one diagonal move forward, left or right, click on the chosen piece, then on it&apos;s landing square.&lt;br/&gt;&lt;br/&gt;You are playing Dark, do the first move.</source>
+        <target>Un simple pas est un déplacement de un en diagonal, en avant, vers la gauche ou la droite, cliquez sur une pièce au choix, puis sur sa case d&apos;arrivée.&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faites le premier mouvement.</target>
+      </trans-unit>
+      <trans-unit id="1675047371672154041" datatype="html">
+        <source>You have to capture when you can, it is the case, click on the piece that must capture and then on its landing square.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, go ahead.</source>
```

Comment:
t'as relu le tuto ? :D 
Respiration, "It is the case *here*"

Reply:
C'EST TOI L'RESPIRIANISTE !